src/style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  color: #000;
```

Comment:
`font-family` and `color` are inherited style. Not need use * for them.
Move it to `body` or `container`.